src/sentry/integrations/vercel/integration.py

```diff
@@ -48,6 +49,8 @@
     aspects={},
 )
 
+internal_integration_overview = "This internal integration was auto-generated during the installation process of your Vercel integration. It is needed to provide the token used to create a release. If this integration is deleted, your Vercel integration will stop working!"
```

Comment:
If someone has a suggestion for a more readable way of defining a long string that doesn't have newline characters I would be very interested :)